ANTsRegistration/ANTsRegistration.py

```diff
@@ -1237,7 +1237,64 @@ def buildTemplate(
             gwtb.SetInitialTemplateImage(initialTemplateImage)
         gwtb.SetPathList(pathList)
 
+        stage = stages[0]
         # TODO: construct pairwise registration instance and set it to gwtb
+        ants_reg = itk.ANTSRegistration[
```

Comment:
Right about the entire code addition here can be shared with pairwise code, and it would probably make sense to put it into its own function.